### PR TITLE
Rename the BBEs With Dots in the Titles

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Rename the BBEs with dots in the titles.
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by NipunaMarcus** [comment]
I think the convention is to put "_" in the file name. So we might have to replace the "-" with "_" and it will be replaced with "-" in the BBE generator.

**Comment by praneesha** [comment]
> I think the convention is to put "_" in the file name. So we might have to replace the "-" with "_" and it will be replaced with "-" in the BBE generator.

Done.

**Comment by praneesha** [comment]
> Also you have to change the index.json inside the examples directory as we have this name there.
> https://github.com/ballerina-platform/ballerina-lang/blob/master/examples/index.json#L807

Done

File: examples/index.json [hunk 1]
```diff
@@ -805,11 +805,11 @@
         "category": "Working over the network",
         "samples": [{
                 "name": "HTTP 1.1 to 2.0 Protocol Switch",
-                "url": "http-1.1-to-2.0-protocol-switch"
+                "url": "http-1_1-to-2_0-protocol-switch"
```
Comment (NipunaMarcus) [inline examples/index.json:808]: here underscore ("_") should be replaced by dash ("-")

File: examples/index.json [hunk 2]
```diff
@@ -805,11 +805,11 @@
...
             },
             {
                 "name": "HTTP 2.0 Server Push",
-                "url": "http-2.0-server-push"
+                "url": "http-2_0-server-push"
```
Comment (NipunaMarcus) [inline examples/index.json:812]: here underscore ("_") should be replaced by dash ("-")

**Comment by NipunaMarcus** [comment]
bbe directory name you should only have dashes hence remove the underscore("_") and replace it with dash("-").
So its like this

examples/bbe<dash>dir/bbe<underscore>file.bal

File: examples/index.json (hunk 1 above)
Comment (praneesha) [reply examples/index.json]: Done.

File: examples/index.json (hunk 2 above)
Comment (praneesha) [reply examples/index.json]: Done

**Comment by praneesha** [comment]
> bbe directory name you should only have dashes hence remove the underscore("_") and replace it with dash("-").
> So its like this
> 
> examples/bbedir/bbefile.bal

Done

**Comment by praneesha** [comment]
Closing as the BBEs are now moved to the "ballerina-distribtuion" repo and this will be fixed in it via [1].

[1] https://github.com/ballerina-platform/ballerina-distribution/pull/501

**Pull request closed without merging**